Package.swift

```diff
@@ -17,6 +17,7 @@ let package = Package(
 		),
 	],
 	dependencies: [
+		// TODO: .package(name: "secp256k1", url: "https://github.com/GigaBitcoin/secp256k1.swift", .upToNextMinor(from: "0.10.0")),
 		.package(url: "https://github.com/tesseract-one/CSecp256k1.swift.git", from: "0.2.0"),
```

Comment:
I'm leaving this dependency change commented out until a later PR where I can work through pod lint issues.